config/_default/menus/menus.en.yaml

```diff
@@ -99,6 +99,12 @@ main_left:
     url: 'https://www.datadoghq.com/product/security-platform/cloud-workload-security/'
     weight: 16
     parent: product
+  - identifier: 'sensitive-data-scanner'
+    name: 'Sensitive Data Scanner'
+    pre: "<i class='icon-sds'></i>"
+    url: 'product/sensitive-data-scanner/'
```

Comment:
i think we need the full absolute url here, since this is linking to corp from docs.

Reply:
good catch. thanks!